lib/apiRequest.js

```diff
@@ -1,48 +1,53 @@
-const fetch = require("node-fetch");
-const getTime = require('./time');
-const generateSignature = require('./generateSignature');
-  
-async function apiRequest(method, path, params, body){  
-  const url = `${process.env.API_URL}${path}${params}`
-  const apiPath = `${process.env.API_PATH}${path}`
+const fetch = require('node-fetch');
+const { getTime } = require('./time');
+const { generateSignature } = require('./generateSignature');
+require('dotenv').config();
+
+async function apiRequest(method, path, params, body) {
+  const url = `${process.env.API_URL}${path}${params}`;
+  const apiPath = `${process.env.API_PATH}${path}`;
 
   const unixTimeResponse = await getTime();
   const unixTime = unixTimeResponse.time;
   const jsonBody = JSON.stringify(body);
   const signedPayload = generateSignature(unixTime, method, apiPath, jsonBody);
 
-  const response = await fetch(url, 
-    url === "/saving/withdrawals"? {
-        method,
-        headers: {
-        "Content-Type": "application/json",
-        "X-AUTH-TOKEN": process.env.API_TOKEN,
-        },
-    }:
-    method === "POST"? {
-        method,
-        headers: {
-        "Content-Type": "application/json",
-        "YAYA-API-KEY": process.env.API_KEY,
-        "YAYA-API-TIMESTAMP": unixTime,
-        "YAYA-API-SIGN": signedPayload,
-        },
-        body: jsonBody,
-    }:
-    {
-        method,
-        headers: {
-        "Content-Type": "application/json",
-        "YAYA-API-KEY": process.env.API_KEY,
-        "YAYA-API-TIMESTAMP": unixTime,
-        "YAYA-API-SIGN": signedPayload,
-        },
-    });
+  const response = await fetch(
+    url,
+    url === '/saving/withdrawals'
+      ? {
+          method,
+          headers: {
```

Comment:
I don't think this works, the postman has wrong headers but all APIs have the same authentication method.
see https://docs.yayawallet.com/hc/main/articles/1699693758-api-authentication

Reply:
> I don't think this works, the postman has wrong headers but all APIs have the same authentication method. see https://docs.yayawallet.com/hc/main/articles/1699693758-api-authentication

@agebeyaw I followed the headers in the collection. Will change it.